src/App.tsx

```diff
@@ -17,7 +17,9 @@ const App = () => {
     const { init: initTrackJS } = useTrackjs();
     const { isDesktop } = useDevice();
     const { redirectToOauth } = useRedirectToOauth();
+    const { initialise: initDerivAnalytics } = useDerivAnalytics();
 
+    initDerivAnalytics();
```

Comment:
[Q] is it possible to add it under initTrackJS? so that any issue in analytics is tracked?